substrate/frame/indices/src/lib.rs

```diff
@@ -232,6 +232,60 @@ pub mod pallet {
 			Self::deposit_event(Event::IndexFrozen { index, who });
 			Ok(())
 		}
+
+		/// Reconsider the deposit reserved for an index.
+		///
+		/// The dispatch origin for this call must be _Signed_ and the signing account must have a
+		/// non-frozen account `index`.
+		///
+		/// The transaction fees is waived if the deposit is changed after reconsideration.
+		///
+		/// - `index`: the index whose deposit is to be reconsidered.
+		///
+		/// Emits `DepositReconsidered` if successful.
+		///
+		/// ## Complexity
+		/// - `O(1)`.
```

Comment:
These complexity annotations are legacy before we had auto-generated weights. You can omit it.